src/Array.ts

```diff
@@ -968,15 +960,15 @@ export const reverse = <A>(as: Array<A>): Array<A> => {
  *
  * assert.deepStrictEqual(rights([right(1), left('foo'), right(2)]), [1, 2])
  *
- * @since 1.0.0
+ * @since 2.0.0
  */
 export const rights = <L, A>(as: Array<Either<L, A>>): Array<A> => {
   const r: Array<A> = []
   const len = as.length
   for (let i = 0; i < len; i++) {
     const a = as[i]
-    if (a.isRight()) {
-      r.push(a.value)
+    if (a._tag === 'Right') {
```

Comment:
is this for perf reasons (instead of using `isRight()`)?

Reply:
to avoid a runtime dependency